src/UI/Internal/RenderConfig.elm

```diff
@@ -0,0 +1,39 @@
+module UI.Internal.RenderConfig exposing (Locale(..), RenderConfig(..), RenderConfigData, string, strings)
+
+import Element
+import I18n.English as LangEnglish
+import I18n.Spanish as LangSpanish
+import I18n.Type as I18n
+
+
+type alias RenderConfigData =
+    { deviceClass : Element.DeviceClass
+    , deviceOrientation : Element.Orientation
+    , locale : Locale
+    }
+
+
+{-| `RenderConfig.RenderConfig` upholds all the information required for the components to apply the responsivity and accessibility changes.
+-}
+type RenderConfig
```

Comment:
I would not make RenderConfig opaque internally, only inside of `UI.RenderConfig.elm`, this way I think it's easy to work with internally.

Reply:
It would be impossible for us to extract it...

Reply:
Why? I didn't get it 🤔

Reply:
The other components are calling `localeTerms`, they have the opaque `RenderConfig`, I would need to add a method somewhere to "de-opaque" it for using `localeTerms`.